com.unity.sg2/Editor/GraphDeltaRegistry/GraphDelta/GraphDataHandler.cs

```diff
@@ -121,7 +121,7 @@ internal void SetData<T>(string layer, T data)
 
         public NodeHandler ToNodeHandler()
         {
-            if(Reader.Element.Header is NodeHeader)
+            if(Reader?.Element?.Header is NodeHeader)
             {
                 return new NodeHandler(ID, Owner, Registry, DefaultLayer);
             }
```

Comment:
Should `GetNode` fail gracefully (returns null), or should it throw an exception? If the latter, we need a `HasNode` or `NodeExists` if we don't already.

Reply:
It looks like elsewhere in the code expects for it to be null-- I'll let Liz decide on this.